app/questionnaire/placeholder_transforms.py

```diff
@@ -30,6 +30,10 @@ def format_list(list_to_format):
 
         return formatted_list
 
+    @staticmethod
+    def remove_empty(list_to_filter):
+        return [item for item in list_to_filter if item]
```

Comment:
In the `first_non_empty_item` we decided that `falsy` values like `False` and `0` are not empty, hence we should be consistent if we are calling the `remove_empty` and not remove `False`/`0`.

Reply:
Also makes it consistent with first_non_empty_item